missing py module pywinrm causing 'make  build-node-image'' to fail building Windows Server 2022 image

**What steps did you take and what happened:**
Building Windows Server 2022 node image, the process fails when running the docker build command within the 'make build-node-image' command when executing the 'make deps-ova' run step. Error comes up warning about missing module pywinrm and the process then errors out as below 

Using default image builder base image library/photon:5.0
Dockerfile:53
--------------------
  51 |     ENV LC_ALL=en_US.UTF-8
  52 |     
  53 | >>> RUN make deps-ova
  54 |     
  55 |     # Make sure packer, ansible and ovftool are installed properly
--------------------
ERROR: failed to solve: process "/bin/bash -c make deps-ova" did not complete successfully: exit code: 2
make: *** [Makefile:75: build-image-builder-container] Error 1


**What did you expect to happen:**
To build the container, run it and create a Windows VM and then template it.

**Is there anything else you would like to add?**
No

**Please tell us about your environment.**

Hypervisor:          vSphere 8u3b
Build machine:    Ubuntu 22.04
VKS Release:       3.3

<table>
<tr>
<td></td>
<th>Value</th>
<th>How to Obtain</th>
</tr>
<tr>
<th>Commit ID</th>
<td><code><!-- Placeholder for the version --></code></td>
<td>Run <code>git log -1</code></td>
</tr>
<tr>
<th>Kubernetes Version</th>
<td><code>v1.32.0+vmware.6-fips</code></td>
<td>Kubernetes version that you are trying to build the image</td>
</tr>
<tr>
<th>OS Type</th>
<td><code>Window Server 2022</code></td>
<td>OS Type and version that you are trying to build the image</td>
</tr>
</table>

--- a/Dockerfile
+++ b/Dockerfile
@@ -25,6 +25,9 @@ RUN for package in unzip git wget build-essential python3-pip jq coreutils opens
 # Install Semver
 RUN pip3 install semver jinja2 jinja2-time
 
+# Install Windows Remote Management 
+RUN pip3 install pywinrm
+
 # Install ovftool
 # TODO: this URL might change or expire so need to look at better way to install it on the container.
 RUN wget https://vdc-download.vmware.com/vmwb-repository/dcr-public/2ee5a010-babf-450b-ab53-fb2fa4de79af/2a136212-2f83-4f5d-a419-232f34dc08cf/VMware-ovftool-4.4.3-18663434-lin.x86_64.zip

